service/src/main/java/bio/terra/workspace/service/workspace/flight/GcpCloudSyncStep.java

```diff
@@ -63,14 +63,15 @@ public StepResult doStep(FlightContext flightContext)
       Policy currentPolicy =
           resourceManagerCow
               .projects()
-              .getIamPolicy(gcpProjectName, new GetIamPolicyRequest())
+              .getIamPolicy(gcpProjectId, new GetIamPolicyRequest())
               .execute();
 
       List<Binding> newBindings = new ArrayList<>();
       // Add all existing bindings to ensure we don't accidentally clobber existing permissions.
       newBindings.addAll(currentPolicy.getBindings());
+      // Add appropriate project-level roles for each WSM IAM role.
       for (Map.Entry<WsmIamRole, String> entry : workspaceRoleGroupsMap.entrySet()) {
```

Comment:
`Map#forEach()` can be easier to read

Reply:
it's a lot nicer having named variables instead of `entry.getKey()`. Thanks!